src/animations/TimingAnimation.js

```diff
@@ -48,4 +48,12 @@ export default class TimingAnimation extends Animation {
   stop() {
     // this._finished && this._finished.setValue(1);
   }
+  static persistenceStateValue = 'frameTime';
+  static getDefaultState() {
+    return {
+      finished: new AnimatedValue(0),
+      time: new AnimatedValue(0),
+      frameTime: new AnimatedValue(0),
```

Comment:
This seems to be the same for all animation types, no?

Reply:
No, it's not